x-pack/test/functional/es_archives/rule_registry/alerts/mappings.json

```diff
@@ -1,7 +1,11 @@
 {
   "type": "index",
   "value": {
-    "index": ".alerts-observability.apm.alerts",
+    "aliases": {
```

Comment:
previously, the rule registry tests were loading this data without running any rules, so there were no existing indices to conflict with. Now there are existing indices that conflict. Updated the data and mappings to better reflect what actual alert data looks like (with the `.internal-*` concrete index names and aliases.